packages/ui/components/form/inputs/Input.tsx

```diff
@@ -411,3 +411,23 @@ export const NumberInput = forwardRef<HTMLInputElement, InputFieldProps>(functio
     />
   );
 });
+
+export const FilterSearchField = forwardRef<HTMLInputElement, InputFieldProps>(function TextField(
+  props,
+  ref
+) {
+  return (
+    <div
+      dir="ltr"
+      className="focus-within:ring-brand-default group relative mx-3 mb-1 mt-2.5 flex items-center rounded-md focus-within:outline-none focus-within:ring-2">
+      <div className="addon-wrapper border-default [input:hover_+_&]:border-emphasis [input:hover_+_&]:border-l-default [&:has(+_input:hover)]:border-emphasis [&:has(+_input:hover)]:border-r-default flex h-7 items-center justify-center rounded-l-md border border-r-0">
+        <Search className="ms-3 h-4 w-4" />
+      </div>
+      <Input
+        ref={ref}
+        className="disabled:bg-subtle disabled:hover:border-subtle !my-0 h-7 rounded-l-none border-l-0 !ring-0 disabled:cursor-not-allowed"
+        {...props}
+      />
+    </div>
+  );
+});
```

Comment:
Should this be here or in filters? Im not too sure myself so just wanted to know what others think

Reply:
This field can be used in other pages such as insights

Reply:
This should be a feature component IMHO. It belongs to features/filters/components.